contributing/how_to_contribute.rst

```diff
@@ -0,0 +1,112 @@
+.. _doc_ways_to_contribute:
+
+How to contribute
+=================
+
+The Godot Engine is free and open-source. Like any community-driven project, we rely on volunteer contributions. 
+On this page we want to showcase the various ways you as users can participate - to help you find the right starting place with the skillset you have.
+Because contrary to popular opinion, we need more than just programmers on the project!
```

Comment:
I think this is a style question, and since the intention is actually less to poke fun at ourselves rather than directly adress that myth I keep hearing ("But I am not a programmer, I can't contribute.") so I would prefer to leave it as is.